packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/index.md

```diff
@@ -0,0 +1,165 @@
+remediations. An _Authenticator_ represents a type, or factor for verifying the identity
+of a user, such as a username and password, or Okta Verify. A _Method_ represents a
+channel for an authenticator, such as using email or SMS for an OTP. An authenticator may
+have multiple methods. A _Remediation_ represents a user action and usually contains most
+of the information for a step. Each response may contain multiple remediations and
+authenticators.
+
+Some common types of user action are represented by a _Capability_. These include
+requesting a new OTP, or a password reset request. Remediations, authenticators, and
+methods may contain capabilities.
+
+A _Form_ inside the remediation represents the user action in a collection of _Field_
+objects. Most fields represent something that's displayed in the UI, either a static
+element, such as a label, or a user input, such as a selection list. The field also
+contains state information, such as whether the associated value is required or secret.
+Options, or lists of choices, are represented by a collection of fields. A field may also
+contain a form which contains more fields.
+
+<div class="common-image-format">
+
+![A diagram showing the SDK objects and their relationships.](/img/mobile-sdk/mobile-idx-objects.png "A diagram that shows the SDK objects for the sign in flow and the relationships between them.")
+
+</div>
+
+## Common parts of the flow
+
+Some parts of the sign-in flow are the same:
+
+- Add the SDK to your app.
+- Configure the SDK.
+- Initialize the client and start the flow.
+- Process the response.
+- Request a token.
+- Sign-out the user.
+
+### Add the SDK
+
+<StackSnippet snippet="adddependency" />
+
+### Create and manage configurations
+
+A _Configuration_ contains the settings used by the SDK to connect to an Okta Application Integration:
+
+| Value         | Description |
+| :------------ | :---------- |
+| Issuer        | The Oauth2 URL for the Okta Org, such as `https://oie-123456.oktapreview.com/oauth2/default`. |
+| Client ID     | The ID of the Okta Application Integration from the Okta Admin console.  |
+| Client secret | An optional shared secret for accessing the Application Integration. |
+| Redirect URI  | A callback URI for launching the mobile app, such as `com.example.oie-123456:/callback`. |
+| Scopes        | A space separated list of the requested access scopes for the app. |
+
+The configuration information in a shipping app is usually static. You can initialize the
+configuration values directly in the code or by reading them in from a file. During
+development you may want to provide a way to edit configuration values.
+
+<StackSnippet snippet="loadingaconfiguration" />
+
+### Start sign-in
+
+Start the sign-in flow by first creating an SDK client using a configuration, and then requesting the first step.
+
+<StackSnippet snippet="initializingsdksession" />
+
+### Process the response
+
+The steps for processing a response are:
```

Comment:
"Check" and "Process" are somewhat non descriptive. I'd recommend we find better alternatives.

Reply:
"Process" is intentionally generic as the actions taken for the response vary considerably, from finding out the user is now signed-in to building a UI from the remediations and authenticators.

I'm open to suggests for "Check". I used it because the step checks for the condition.